doc/release/release_dev.rst

```diff
@@ -36,8 +36,9 @@ New functions:
 - maximum cut heuristics
 - equivalence_classes
 - dedensification
+- random_ordered_tree
```

Comment:
AFAICT, #4294 modifies the implementation of `random_tree`, but doesn't actually introduce a new function called `random_ordered_tree`. Maybe adding a sentence about the `create_using` parameter to the Improvement bullet for 4294 would be better?

Reply:
Got it!   This makes sense...